src/utils/feature-flag-utils.ts

```diff
@@ -9,12 +9,12 @@ declare const window: any
  */
 const defaultFlagSet: LDFlagSet = {
   'alteration-ui-enabled': false,
+  'banner-text': '', // by default, there is no banner text
   'change-ui-enabled': false,
   'conversion-ui-enabled': false,
-  'correction-ui-enabled': false,
   'restoration-ui-enabled': false,
   'sentry-enable': false, // by default, no sentry logs
-  'banner-text': '' // by default, there is no banner text
+  'supported-correction-entities': []
```

Comment:
In case LD can't be reached (esp in Prod), this is a safe fallback.

If you forget to add the dev LD key to your `.env` file, you get this value too 😁 Otherwise, your local and dev should have the same flags.